dev-libs/mpvqt/mpvqt-1.0.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KFMIN=5.245.0
+QTMIN=6.5.0
+inherit ecm kde.org
+
+if [[ ${KDE_BUILD_TYPE} = release ]]; then
+	SRC_URI="mirror://kde/stable/${PN}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
+fi
+
+DESCRIPTION="MpvQt is a libmpv wrapper for QtQuick2 and QML"
+HOMEPAGE="https://invent.kde.org/libraries/mpvqt"
+
+LICENSE="|| ( GPL-2 GPL-3 LGPL-3 LGPL-2.1 ) CC-BY-SA-4.0 MIT BSD"
+SLOT="6"
+IUSE=""
+
+DEPEND="
+	media-video/mpv:=[libmpv]
+	>=dev-qt/qtdeclarative-${QTMIN}:6
+	>=kde-frameworks/extra-cmake-modules-${KFMIN}
```

Comment:
ecm should be a BDEPEND but eclass should handle this for you anyway

Reply:
D'oh right, fixed